src/components/app/AppHeader.vue

```diff
@@ -1,20 +1,28 @@
-<!-- The main header for the frontend -->
-
 <template id="app-header">
-  <v-app-bar flat height="124">
+  <v-app-bar flat height="124" class="pa-0 ma-0">
     <template v-slot:image>
-      <v-container fluid class="pt-0">
-        <v-img height="110" cover :src="banner" :lazy-src="banner" />
-        <v-img cover :src="foot" :lazy-src="foot" />
+      <!-- in order to have no padding on the top image
+        you need to set padding and margin to be 0 and then 
+        set fluid to true specifically in that order. Appears to
+        be a vuetify quirk 
+      -->
+      <v-container class="pa-0 ma-0" fluid>
+        <div 
+          v-if="customBannerColor" 
+          :style="{ backgroundColor: customBannerColor, height: '110px' }">
+        </div>
+        <v-img v-else height="110px" cover :src="banner" :lazy-src="banner" />
+
+        <v-img v-if="!disableSeparator"  cover :src="foot" :lazy-src="foot" />
       </v-container>
     </template>
 
     <template v-slot:default>
       <v-responsive width="90%" max-width="1130" height="100%" class="mx-auto">
         <v-container fluid>
-          <v-row align="start" fill-height>
-            <a :href="wis2" title="wis2box">
-              <img :src="logo" class="pl-6 my-3" />
+          <v-row align="center" class="fill-height">
+            <a :href="wis2" title="wis2box" class="logo-container">
```

Comment:
@maaikelimper should we still hyperlink the logo to https://community.wmo.int/activity-areas/wis/wis2-implementation or should also be an env variable?

Reply:
Could also hyperlink to homepage of wis2box-ui and then remove the nav-bar

Reply:
@webb-ben remove link community.wmo.int and replace with hyperlink to homepage of wis2box-ui and remove nav-bar

Reply:
the link to wis2box-documentation currently in the nav-bar could move to the bottom:

So instead of 
"Powered by [wis2box-ui](https://github.com/wmo-im/wis2box-ui) 1.0.0-beta8"

We could have
"Powered by [wis2box](https://docs.wis.wis2box.wmo.int) , [wis2box-ui](https://github.com/wmo-im/wis2box-ui) 1.0.0-beta8,

Reply:
OK, reading the latest commit I agree with @C-Loftus that the remaining changes can move to a separate PR